src/device/router_device_worker.erl

```diff
@@ -885,6 +885,7 @@ handle_info(
                 DownlinkPacket,
                 Region
             ),
+            ok = maybe_send_queue_update(Device1, State),
```

Comment:
Idk about this one as we do not actually update the queue but rather potentially remove a downlink from it

Reply:
When we send a downlink they don't want to see it in the side panel any more.

Reply:
Unless it's a confirmed down. But once it's out of the queue, they want that shown on the frontend

Reply:
sold